tests/suites/casper/flow.js

```diff
@@ -0,0 +1,55 @@
+/*global casper*/
```

Comment:
When I have used  "jshint" (thanks to Juga Paazmaya) a lot of errors appears.
So I try to do everything to pass them.

erasing blank chars, count functions, add comments ...